watchface

```diff
@@ -0,0 +1,406 @@
+            then "${action}"face "${watchface}" "${newface}"
+            else echo "No name entered"
+            fi
+        elif [ "${action}" = "deploy" ] ; then
+            local first=true
+            for wf in ${watchface}; do
+                "${action}"face "${wf}" "${first}"
+                first=false
+            done
+            if [ "${REBOOT}" = "true" ] ; then
+                rebootWatch
+            fi
+        else
+            "${action}"face "${watchface}"
+        fi
+    else
+        echo "Canceled"
+    fi
+}
+
+function textmenu {
+    local args
+    local watchface
+    local newface
+    if ! action=$("${MENUPROGRAM}" --title "Select action" --clear --radiolist \
+        "Select action" 0 0 3 \
+        "deploy" "Deploy watchface to watch" ON \
+        "clone" "Clone watchface to new name" OFF \
+        "test" "Test watchface on computer" OFF \
+        3>&1 1>&2 2>&3); then
+        exit
+    fi
+    case $action in 
+        deploy)
+            declare -a args=("--title" "Watchfaces" "--clear" "--checklist" "Choose watchfaces:" 25 78 15 "--" )
+            for wf in "${WATCHFACES[@]}" ; do
+                args+=("${wf}" "" OFF)
+            done
+            if ! watchface=$("${MENUPROGRAM}" "${args[@]}" 3>&1 1>&2 2>&3); then
+                exit
+            fi
+            ;;
+        clone|test)
+            declare -a args=("--title" "Watchfaces" "--clear" "--menu" "Choose a watchface:" 25 78 15 "--" )
+            for wf in "${WATCHFACES[@]}" ; do
+                args+=("${wf}" "")
+            done
+            if ! watchface=$("${MENUPROGRAM}" "${args[@]}" 3>&1 1>&2 2>&3); then
+                exit
+            fi
+            ;;
+    esac
+    if ("${MENUPROGRAM}" --clear --title "Confirming" --yesno "${action} ${watchface}" 0 0); then
+        echo "OK!"
+        if [ "${action}" = "clone" ] ; then
+            if newface=$("${MENUPROGRAM}" \
+                --title "Cloning ${watchface}" --clear \
+                --inputbox "Enter name of new watchface:" \
+                8 50 "newface" \
+                3>&1 1>&2 2>&3)
+            then
+                "${action}"face "${watchface}" "${newface}"
+            else
+                echo "No name entered"
+            fi
+        elif [ "${action}" = "deploy" ] ; then
+            local first=true
+            for wf in ${watchface}; do
+                "${action}"face "${wf}" "${first}"
+                first=false
+            done
+            if [ "${REBOOT}" = "true" ] ; then
+                rebootWatch
+            fi
+        else
+            "${action}"face "${wf}"
+        fi
+    else
+        echo "Canceled"
+    fi
+}
+
+# These are the defaults for SSH access
+WATCHPORT=22
+WATCHADDR=192.168.2.15
+# These are the defaults for local QEMU target
+QEMUPORT=2222
+QEMUADDR=localhost
+# Assume no ADB unless told otherwise
+ADB=false
+# Assume no GUI unless told otherwise
+GUI=false
+# we're not done until we say we are
```

Comment:
Let's keep case consistent, all other comments have a capital letter, let's use one here too ;)

Reply:
Right!  I've also rewritten the comment to be a bit more helpful in understanding the code.